language-server/modules/langserver-core/src/test/resources/codeaction/ignore-unused-var/source/ignore_unused_var2.bal

```diff
@@ -0,0 +1,7 @@
+function calc(float a, float b) {
+    int[] arr = [1, 2, 3];
+    int total = 0;
+    foreach int i in arr {
```

Comment:
I'm getting a syntax error inside the `foreach`.  Is this expected?

<img width="347" alt="Screenshot 2021-11-03 at 09 49 33" src="https://user-images.githubusercontent.com/27485094/140009650-f5d2d69a-a09a-4231-bfac-4f2613299041.png">

Reply:
This was due to the diagnostics in the Compiler Toolkit. Therefore, marking as resolved.

Reply:
What is the expected behaviour when there are 2 unused variables?

<img width="523" alt="Screenshot 2021-11-03 at 10 18 50" src="https://user-images.githubusercontent.com/27485094/140011401-fd4e0339-aea6-412c-bc55-d9037547e067.png">

Reply:
I have the same question as well. Need to check with the front end team.

Reply:
Updated the codeaction to ignore list binding pattern or mapping binding pattern in cases as above.